### PR TITLE
[Behat][Promotion] Fixed discount on taxon scenarios

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets |
| License         | MIT
| Doc PR          |

Scenarios are based on *Receiving fixed discount on cart* and *Receiving percentage discount on products from specific taxon*.

### CODE REVIEW

File: features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        Then its price should be decreased by "€10.00"
+        And my cart total should be "€90.00"
+        And my discount should be "-€10.00"
+
+    @todo
+    Scenario: Receiving percentage discount on a multiple items from specific taxon
+        When I add 3 products "PHP T-Shirt" to the cart
+        Then their price should be decreased by "€10.00"
+        And my cart total should be "€270.00"
+        And my discount should be "-€20.00"
```
Comment (pamil) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: `-€30.00`?

File: features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    Scenario: Receiving fixed discount equal to the items total of my cart
+        Given there is a promotion "Christmas Sale"
+        And it gives "€20.00" off every product classified as "Mugs"
+        When I add 3 products "PHP Mug" to the cart
+        Then their price should be decreased by "€60.00"
+        And my cart total should be "€0.00"
+        And my discount should be "-€60.00"
+
+    @todo
+    Scenario: Receiving fixed discount equal to the items total of my cart even if the discount is bigger than the items total
```
Comment (pamil) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: Invalid scenario, PHP mug price = €20, discount on every item = €20, cart total = €0, discount should be bigger than €20.

File: features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature (hunk 1 above)
Comment (Zales0123) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: :+1:

File: features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store classifies its products as "T-Shirts" and "Mugs"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And it belongs to "T-Shirts"
+        And the store has a product "PHP Mug" priced at "€20.00"
+        And it belongs to "Mugs"
+        And there is a promotion "T-Shirts promotion"
+        And it gives "€10.00" off every product classified as "T-Shirts"
```
Comment (lchrusciel) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: off **for** every

Comment (TheMadeleine) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: **on** every product

Comment (lchrusciel) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: :+1:

File: features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature (hunk 2 above)
Comment (Zales0123) [inline features/promotion/receiving_fixed_discount_on_products_from_specific_taxon.feature]: :+1:

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**